components/footer.jsx

```diff
@@ -4,6 +4,10 @@ import { IntlMixin, FormattedHTMLMessage } from 'react-intl';
 var Footer = React.createClass({
   mixins: [IntlMixin],
   render: function() {
+    var firefox_footer = this.getIntlMessage('firefox_footer');
+    if (window.location.pathname.indexOf('thunderbird') !== -1) {
```

Comment:
We could have a different footer, and apply that component only to the TB-specific layouts. It just seemed like overkill to have a different component just for one string change. That is why we decided to query the pathname.